theClanless/src/main/java/theClanless/cards/QuickJab.java

```diff
@@ -0,0 +1,71 @@
+package theClanless.cards;
+
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.common.DamageAction;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.localization.CardStrings;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import theClanless.characters.TheClanless;
+import theClanless.theClanlessMod;
+
+import static theClanless.theClanlessMod.makeCardPath;
+
+public class QuickJab extends AbstractDynamicCard {
+
+    // public static final String ID = DefaultMod.makeID(${NAME}.class.getSimpleName()); // USE THIS ONE FOR THE TEMPLATE;
+    public static final String ID = theClanlessMod.makeID("QuickJab"); // DELETE THIS ONE.
+    public static final String IMG = makeCardPath("QuickJab.png");// "public static final String IMG = makeCardPath("${NAME}.png");
+    // This does mean that you will need to have an image with the same NAME as the card in your image folder for it to run correctly.
+
+
+    // /TEXT DECLARATION/
+
+
+    // STAT DECLARATION
+    private static final CardStrings cardStrings = CardCrawlGame.languagePack.getCardStrings(ID);
+    public static final String DESCRIPTION = cardStrings.DESCRIPTION;
+    public static final String UPGRADE_DESCRIPTION = cardStrings.UPGRADE_DESCRIPTION;
+
+    private static final CardRarity RARITY = CardRarity.SPECIAL; //  Up to you, I like auto-complete on these
+    private static final CardTarget TARGET = CardTarget.ENEMY;  //   since they don't change much.
+    private static final CardType TYPE = CardType.ATTACK;       //
+    public static final CardColor COLOR = TheClanless.Enums.COLOR_CLANLESSRED;
+
+    private static final int COST = 0;  // COST = ${COST}
+
+    private static final int DAMAGE = 5;    // DAMAGE = ${DAMAGE}
+    private static final int UPGRADE_PLUS_DMG = 3;  // UPGRADE_PLUS_DMG = ${UPGRADED_DAMAGE_INCREASE}
+
+    // /STAT DECLARATION/
+
+
+    public QuickJab() { // public ${NAME}() - This one and the one right under the imports are the most important ones, don't forget them
+        super(ID, IMG, COST, TYPE, COLOR, RARITY, TARGET);
+        this.damage = this.baseDamage = DAMAGE;
+        this.exhaust = true;
+        this.tags.add(CardTags.STRIKE);
```

Comment:
Why is this a STRIKE?

Reply:
because it's ADDITIONAL! (is there anything besides perfect strike that deals with "strike?")

Reply:
The practice dummy relic that gives all strikes +3 damage. Only things that are "Strikes" should have that tag.

Reply:
Well yes. I consider it a strike.